lte/gateway/c/core/oai/tasks/amf/amf_smf_send.cpp

```diff
@@ -35,16 +35,23 @@ extern "C" {
 #include "lte/gateway/c/core/oai/tasks/amf/amf_app_timer_management.h"
 #include "lte/gateway/c/core/oai/tasks/amf/amf_common.h"
 #include "lte/gateway/c/core/oai/tasks/nas/api/mme/mme_api.h"
+#include "lte/gateway/c/core/oai/tasks/amf/amf_app_defs.h"
 
 using magma5g::AsyncM5GMobilityServiceClient;
 using magma5g::AsyncSmfServiceClient;
 
 namespace magma5g {
 #define IMSI_LEN 15
 #define AMF_CAUSE_SUCCESS 1
-const uint32_t MAX_UE_PDU_SESSION_LIMIT = 15;
+const uint32_t MAX_UE_PDU_SESSION_LIMIT                             = 15;
+const uint32_t MAX_UE_INITIAL_PDU_SESSION_ESTABLISHMENT_REQ_ALLOWED = 5;
 
 int amf_max_pdu_session_reject(amf_ue_ngap_id_t ue_id, ULNASTransportMsg* msg);
+int amf_pdu_session_establishment_reject(
```

Comment:
Can we have these definitions in another header files like :  "include/amf_smf_packet_handler.h"

Reply:
created new header file as specified.